--- gnustep-gui-0.9.1/Tools/make_services.m.orig	2003-10-10 05:55:11.000000000 +0200
+++ gnustep-gui-0.9.1/Tools/make_services.m	2003-12-29 16:08:38.987141624 +0100
@@ -199,6 +199,12 @@
   usrRoot = [NSSearchPathForDirectoriesInDomains(NSUserDirectory,
     NSUserDomainMask, YES) lastObject];
 
+  /* don't SEGV if ~/GNUstep doesn't exist */
+  if(usrRoot == nil)
+    {
+      exit(EXIT_FAILURE);
+    }
+
   mgr = [NSFileManager defaultManager];
   if (([mgr fileExistsAtPath: usrRoot isDirectory: &isDir] && isDir) == 0)
     {
