diff -ur gnustep-gui-0.8.7-orig/Source/Functions.m gnustep-gui-0.8.7/Source/Functions.m
--- gnustep-gui-0.8.7-orig/Source/Functions.m	2003-06-13 09:01:10.000000000 -0600
+++ gnustep-gui-0.8.7/Source/Functions.m	2003-07-24 17:47:22.000000000 -0600
@@ -566,9 +566,9 @@
 			       up_sides, grays, 6);
     }
 
-  DPSsetgray(ctxt, NSLightGray);
-  DPSrectfill(ctxt, NSMinX(rect), NSMinY(rect), 
-	      NSWidth(rect), NSHeight(rect));
+//  DPSsetgray(ctxt, NSLightGray);
+//  DPSrectfill(ctxt, NSMinX(rect), NSMinY(rect), 
+//	      NSWidth(rect), NSHeight(rect));
 }
 
 void
@@ -602,9 +602,9 @@
       DPSrectfill(ctxt, NSMaxX(aRect) - 2., NSMaxY(aRect) - 2., 1., 1.);
     }
 
-  DPSsetgray(ctxt, NSLightGray);
-  DPSrectfill(ctxt, NSMinX(rect), NSMinY(rect), 
-	      NSWidth(rect), NSHeight(rect));
+//  DPSsetgray(ctxt, NSLightGray);
+//  DPSrectfill(ctxt, NSMinX(rect), NSMinY(rect), 
+//	      NSWidth(rect), NSHeight(rect));
 }
 
 void 
@@ -630,9 +630,9 @@
 			       up_sides, grays, 8);
     }
 
-  DPSsetgray(ctxt, NSLightGray);
-  DPSrectfill(ctxt, NSMinX(rect), NSMinY(rect), 
-	      NSWidth(rect), NSHeight(rect));
+//  DPSsetgray(ctxt, NSLightGray);
+//  DPSrectfill(ctxt, NSMinX(rect), NSMinY(rect), 
+//	      NSWidth(rect), NSHeight(rect));
 }
 
 void 
@@ -658,9 +658,9 @@
 			       up_sides, grays, 8);
     }
 
-  DPSsetgray(ctxt, NSWhite);
-  DPSrectfill(ctxt, NSMinX(rect), NSMinY(rect), 
-	      NSWidth(rect), NSHeight(rect));
+//  DPSsetgray(ctxt, NSWhite);
+//  DPSrectfill(ctxt, NSMinX(rect), NSMinY(rect), 
+//	      NSWidth(rect), NSHeight(rect));
 }
 
 void 
@@ -700,9 +700,9 @@
 			       up_sides, grays, 6);
     }
 
-  DPSsetgray(ctxt, NSLightGray);
-  DPSrectfill(ctxt, NSMinX(rect), NSMinY(rect), 
-	      NSWidth(rect), NSHeight(rect));
+//  DPSsetgray(ctxt, NSLightGray);
+//  DPSrectfill(ctxt, NSMinX(rect), NSMinY(rect), 
+//	      NSWidth(rect), NSHeight(rect));
 }
 
 void 
diff -ur gnustep-gui-0.8.7-orig/Source/NSApplication.m gnustep-gui-0.8.7/Source/NSApplication.m
--- gnustep-gui-0.8.7-orig/Source/NSApplication.m	2003-06-13 09:01:11.000000000 -0600
+++ gnustep-gui-0.8.7/Source/NSApplication.m	2003-07-24 17:48:30.000000000 -0600
@@ -221,7 +221,27 @@
 	  _(@"Backend at path %@ doesn't contain the GSBackend class"), path);
 	[backend initializeBackend];
       }
-
+ 
+       /* hack to load arbitrary bundles - Alexander Malmberg 2002-05-09 */
+       {
+           NSUserDefaults *defs=[NSUserDefaults standardUserDefaults];
+           NSString *s=[defs stringForKey: @"GUIThemeBundles"];
+           NSArray *a=[s componentsSeparatedByString: @" "];
+           int i,c=[a count];
+           NSLog(@"try loading '%@'",s);
+           for (i=0;i<c;i++)
+           {
+              NSLog(@"try '%@'\n",[a objectAtIndex: i]);
+              NSBundle *b=[NSBundle bundleWithPath: [a objectAtIndex: i]];
+              if (!b)
+              {
+                  NSLog(@"Couldn't load theme bundle '%@'!",[a objectAtIndex: i]);
+                  continue;
+              }
+              [[[b principalClass] alloc] init];
+           }
+       }
+ 
 #else
       /* GSBackend will be in a separate library, so use the runtime
 	 to find the class and avoid an unresolved reference problem */
Only in gnustep-gui-0.8.7/Source: NSApplication.m~
Only in gnustep-gui-0.8.7/Source: NSApplication.m.orig
diff -ur gnustep-gui-0.8.7-orig/Source/NSColor.m gnustep-gui-0.8.7/Source/NSColor.m
--- gnustep-gui-0.8.7-orig/Source/NSColor.m	2003-06-13 09:01:11.000000000 -0600
+++ gnustep-gui-0.8.7/Source/NSColor.m	2003-07-24 17:47:22.000000000 -0600
@@ -698,6 +698,13 @@
   return systemColorWithName(@"windowFrameTextColor");
 }
 
++ (void) setColor: (NSColor*) color forKey: (NSString*) key
+{
+        [systemColors setColor: color forKey: key];
+	[[systemDict objectForKey: key] recache];
+}
+
+
 ////////////////////////////////////////////////////////////
 //
 // Instance methods
