--- gstep-0.6.0/gui/Source/GNUmakefile.cls	Tue Sep  7 23:01:23 1999
+++ gstep-0.6.0/gui/Source/GNUmakefile	Sun Sep 19 00:26:02 1999
@@ -137,12 +137,24 @@
 
 libgnustep-gui_HEADER_FILES = \
 AppKit/AppKit.h \
+AppKit/AppKitExceptions.h \
+AppKit/DPSOperators.h \
+AppKit/GMAppKit.h \
+AppKit/GMArchiver.h \
+AppKit/GSMethodTable.h \
+AppKit/GSPasteboardServer.h \
+AppKit/GSServicesManager.h \
+AppKit/GSTrackingRect.h \
+AppKit/GSWraps.h \
+AppKit/IMConnectors.h \
+AppKit/IMCustomObject.h \
+AppKit/IMLoading.h \
 AppKit/NSActionCell.h \
 AppKit/NSAffineTransform.h \
 AppKit/NSApplication.h \
 AppKit/NSAttributedString.h \
-AppKit/NSBitmapImageRep.h \
 AppKit/NSBox.h \
+AppKit/NSBitmapImageRep.h \
 AppKit/NSBrowser.h \
 AppKit/NSBrowserCell.h \
 AppKit/NSButton.h \
@@ -156,6 +168,7 @@
 AppKit/NSColorPanel.h \
 AppKit/NSColorPicker.h \
 AppKit/NSColorPicking.h \
+AppKit/NSColorPrivate.h \
 AppKit/NSColorWell.h \
 AppKit/NSControl.h \
 AppKit/NSCursor.h \
@@ -186,8 +199,8 @@
 AppKit/NSMenuItem.h \
 AppKit/NSMenuItemCell.h \
 AppKit/NSMenuView.h \
-AppKit/NSNibLoading.h \
 AppKit/NSNibDeclarations.h \
+AppKit/NSNibLoading.h \
 AppKit/NSOpenPanel.h \
 AppKit/NSPageLayout.h \
 AppKit/NSPanel.h \
@@ -209,6 +222,7 @@
 AppKit/NSSelection.h \
 AppKit/NSSlider.h \
 AppKit/NSSliderCell.h \
+AppKit/NSText.h \
 AppKit/NSSpellChecker.h \
 AppKit/NSSpellProtocol.h \
 AppKit/NSSpellServer.h \
@@ -217,10 +231,9 @@
 AppKit/NSTabView.h \
 AppKit/NSTabViewItem.h \
 AppKit/NSTableColumn.h \
+AppKit/NSTableHeaderCell.h \
 AppKit/NSTableHeaderView.h \
-AppKit/NSTableHeaderCell.h  \
 AppKit/NSTableView.h \
-AppKit/NSText.h \
 AppKit/NSTextAttachment.h \
 AppKit/NSTextContainer.h \
 AppKit/NSTextField.h \
@@ -230,14 +243,8 @@
 AppKit/NSView.h \
 AppKit/NSWindow.h \
 AppKit/NSWorkspace.h \
-AppKit/GSTrackingRect.h \
-AppKit/nsimage-tiff.h \
-AppKit/GSMethodTable.h \
-AppKit/DPSOperators.h \
 AppKit/PSOperators.h \
-AppKit/GSPasteboardServer.h \
-AppKit/GSServicesManager.h \
-AppKit/GSWraps.h
+AppKit/nsimage-tiff.h
 
 -include GNUmakefile.preamble
 
