diff -ur gnustep-gui-0.24.0/Source/GSImageMagickImageRep.m gnustep-gui-0.24.0-im7/Source/GSImageMagickImageRep.m
--- gnustep-gui-0.24.0/Source/GSImageMagickImageRep.m	2011-07-13 05:34:38.000000000 +0200
+++ gnustep-gui-0.24.0-im7/Source/GSImageMagickImageRep.m	2016-12-25 18:35:02.879438129 +0100
@@ -43,7 +43,7 @@
 
 #if HAVE_IMAGEMAGICK
 
-#include <magick/MagickCore.h>
+#include <MagickCore/MagickCore.h>
 
 @implementation GSImageMagickImageRep 
 
@@ -82,11 +82,11 @@
       NSSize res;
       if (image->units == PixelsPerCentimeterResolution)
 	{
-	  res = NSMakeSize(image->x_resolution * 2.54, image->y_resolution * 2.54);
+	  res = NSMakeSize(image->resolution.x * 2.54, image->resolution.y * 2.54);
 	}
       else
 	{
-	  res = NSMakeSize(image->x_resolution, image->y_resolution);
+	  res = NSMakeSize(image->resolution.x, image->resolution.y);
 	}
 
       if (res.width > 0 && res.height > 0)
@@ -117,11 +117,12 @@
 
   ExceptionInfo *exception = AcquireExceptionInfo();
   ImageInfo *imageinfo = CloneImageInfo(NULL);
+  const ColorInfo *colorinfo = GetColorInfo("none", exception);
   Image *images, *image;
   
   // Set the background color to transparent
   // (otherwise SVG's are rendered against a white background by default)
-  QueryColorDatabase("none", &imageinfo->background_color, exception);
+  imageinfo->background_color = colorinfo->color;
 
   images = BlobToImage(imageinfo, [data bytes], [data length], exception);
 
